symfony/plugins/orangehrmRecruitmentPlugin/Controller/AddJobVacancyController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace OrangeHRM\Recruitment\Controller;
+
+use OrangeHRM\Core\Controller\AbstractVueController;
+use OrangeHRM\Core\Vue\Component;
+use OrangeHRM\Framework\Http\Request;
+
+class AddJobVacancyController extends AbstractVueController
+{
+
+    public function preRender(Request $request): void
+    {
+        $component = new Component('add-job-vacancy');
+        $this->setComponent($component);
+    }
+}
```

Comment:
add empty line end of the file,
Other controller files also have same issue

Reply:
all done